README.md

```diff
@@ -1,18 +1,29 @@
-#SPHARM-PDM Toolbox
-
-##What is it?
+<img src="Modules/Scripted/ShapeAnalysisModule/Resources/Icons/ShapeAnalysisModule2.png" width="1024">
 
+Introduction
+------------
 Shape analysis has become of increasing interest to the medical community due to its potential to precisely locate morphological changes between healthy and pathological structures. SPHARM-PDM is a tool that computes point-based models using a parametric boundary description for the computing of Shape analysis.
 
 The point-based models computed with the SPHARM-PDM tool can be used in combination with the also UNC designed statistical tool shapeAnalysisMANCOVA to perform quantitative morphological assessment of structural changes at speciﬁc locations.
 
-SPHARM-PDM is now available as a 3D Slicer extension (http://www.slicer.org)
+SPHARM-PDM is now available as a 3D Slicer extension (http://www.slicer.org) and as a SlicerSALT module (https://bpaniagua.github.io/slicersalt.github.io/)
 
-##License
+Prerequisites
+-------------
+On 3D Slicer and SlicerSALT, Shape Analysis Module uses: 
+* Shape Population Viewer extension which allows to display the outputs generated by Shape Analysis Module. 
```

Comment:
+* Shape Population Viewer extension displays the outputs generated by Shape Analysis Module.